dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -712,6 +712,10 @@ error.invalidPhaseName=CWWKE0954E: The "{0}" checkpoint phase specified is empty
 error.invalidPhaseName.explanation=An unknown phase was specified as the value of the checkpoint option.
 error.invalidPhaseName.useraction=Review the documentation to determine valid phase names. Reissue the command using a valid checkpoint phase.
 
+error.set.securitymanager.jdk18=CWWKE0955E: Java Security was requested in bootstrapping.properties while using JDK "{0}", however this option is no longer valid when using Java 18 and later.
+error.set.securitymanager.jdk18.explanation= As part of the deprecation of Java Security as specified by JEP 411, Liberty server no longer supports Java Security with Java runtime versions 18 and later.
```

Comment:
Change to "As part of the deprecation of the Java Security Manager, as specified by JEP 411, Liberty no longer supports enabling the Java Security Manager when using Java 18 or later."